pyvista/plotting/plotter.py

```diff
@@ -72,9 +72,6 @@
 from .text import TextProperty
 from .texture import numpy_to_texture
 from .themes import Theme
-from .tools import normalize  # noqa: F401
-from .tools import opacity_transfer_function  # noqa: F401
-from .tools import parse_font_family  # noqa: F401
```

Comment:
I think these were added in #4639. Not sure if or why the import needed though...

Reply:
Actually, it was added in #1962 for an example, not sure if it's still needed, will add back if CI fails